tests/integration/cli/test_pm.py

```diff
@@ -103,3 +103,100 @@ def test_compile_dependency(ape_cli, runner, project):
     result = runner.invoke(ape_cli, ["pm", "compile", name])
     assert result.exit_code == 0, result.output
     assert f"Package '{name}' compiled." in result.output
+
+
+# Remove tests
```

Comment:
one thing you can do that is a good way to document tests is actually move these parts into the "doc str" of the tests.. I think pytest integrates nicer this way (or at least i see other projects do that).

so each test can document the cases it is covering individually instead of altogether up here.

Reply:
example of what i mean:

```python
def test_a_thing():
    """
    This test tests testing test cases test ya this is what i mean.
    """

```

Reply:
noted

Reply:
The problem is that I'm not able to test the remove feature using tests. As to remove the package first I have to install it. How can I emulate this using pytest ?

Reply:
You can run the code to this, basically do what the `add` tests do but treat it as setup instead of a test, like this:

```python
def test_remove():
    project = "with-contracts"
    path = Path(__file__).parent / "projects" / project
    # Ensure package exists before removing
    runner.invoke(ape_cli, ["pm", "install", path.as_posix(), "--name", project])
    
    # Now do your test, assuming the command above succeeded (it is tested separately so that is ok)
```

Reply:
You can even move all of that into a fixture:

```python
@pytest.fixture
def dependency_that_exists():
    runner.invoke(ape_cli, ["pm", "install", path.as_posix(), "--name", project])
    return project.dependencies[whatever]
```